### PR TITLE
Dla gui

### PR DESCRIPTION
**Pull request by profxj**
Edits to xfitdlagui and igmguesses.

Failing Travis due to a scipy issue (unrelated)

### CODE REVIEW

File: xastropy/xguis/abskingui.py [hunk 1]
```diff
@@ -44,7 +44,7 @@ class AbsKinGui(QtGui.QDialog):
     """ GUI to analyze absorption lines for future kinematic analysis
     """
     def __init__(self, ispec, z=None, parent=None, llist=None, norm=True,
-                 vmnx=[-300., 300.]*u.km/u.s, abs_sys=None, outfil='dum_kin.json',
+                 vmnx=[-300., 300.]*u.km/u.s, abs_sys=None, outfil=None,
                  sel_wv=None, name=''):
         """
         spec : Filename or Spectrum1D
```
Comment (ntejos) [inline xastropy/xguis/abskingui.py]: Please add docstring for missing parameters and update naming convention when relevant.

File: xastropy/xguis/abskingui.py [hunk 2]
```diff
@@ -217,10 +222,10 @@ def main(*args, **kwargs):
...
     import sys
     import argparse
-    from specutils import Spectrum1D
 
-    parser = argparse.ArgumentParser(description='Parse for AbsKingGui')
+    parser = argparse.ArgumentParser(description='Parse for AbsKinGui')
     parser.add_argument("file", type=str, help="Spectral file")
+    parser.add_argument("-sysfile", type=str, help="System JSON file")
     parser.add_argument("-zsys", type=float, help="System Redshift")
     parser.add_argument("-outfil", type=str, help="Output filename")
```
Comment (ntejos) [inline xastropy/xguis/abskingui.py]: Please mention that the output file is a JSON file.

Comment (profxj) [inline xastropy/xguis/abskingui.py]: This GUI is now in linetools.  I'll remove it from xastropy sooner or later.

**Comment by profxj** [comment]
merging

**Pull request merged**